### DISCUSSION TITLE
Bug: watchers always same as stargazers_count in API /user/starred

### DISCUSSION THREAD

**Discussion by conghaonet** [community/community#24627] [Apps, API and Webhooks]
<p>Version: v3</p>
<p>API:：/user/starred</p>
<p>URI：/user/starred?page=1&amp;sort=created&amp;direction=des</p>
<p>response:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/4d52f005638379c2f4a3ab539c58f244d95142b9.png" data-download-href="/uploads/short-url/b22spdVvA5HSEXzTYXEodrDD9ep.png?dl=1" title="snapshot.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/4373795/181085732-95877bbd-684a-4cd6-8d8b-8e6105b99a48.png" alt="snapshot.png" data-base62-sha1="b22spdVvA5HSEXzTYXEodrDD9ep" width="350" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/4d52f005638379c2f4a3ab539c58f244d95142b9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">snapshot.png</span><span class="informations">636×906 28.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/conghaonet">@conghaonet</a>,</p>
<p>That’s expected behavior since “starring” was called “watching” in the past, so to prevent breaking applications – that hasn’t changed when starring was introduced.</p>
<p><a href="https://developer.github.com/v3/activity/starring/#starring-vs-watching" rel="nofollow noopener">https://developer.github.com/v3/activity/starring/#starring-vs-watching</a></p>
<p>I hope this helps!</p>

**Comment by conghaonet** [comment]
<p>Hi <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>,</p>
<p>Thank you for your reply!</p>

**Comment by jinyeqing** [comment]
<p>I used the V3 API, but there is no subscriber_count field in the returned data. What can I do?</p>
<p>The photo is as follows：</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/5a518f4c160a88afe2419972d531c4cc11b00da0.png" data-download-href="/uploads/short-url/cSZIdXvUsiQSifithbYaMLU59kI.png?dl=1" title="WechatIMG525.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/14799589/181085733-23d91a3a-c48c-4f95-a5de-db0b9f7d33cb.png" alt="WechatIMG525.png" data-base62-sha1="cSZIdXvUsiQSifithbYaMLU59kI" width="689" height="295" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/5a518f4c160a88afe2419972d531c4cc11b00da0_2_689x295.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/5a518f4c160a88afe2419972d531c4cc11b00da0_2_1033x442.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/5/5a518f4c160a88afe2419972d531c4cc11b00da0_2_1378x590.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/5a518f4c160a88afe2419972d531c4cc11b00da0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">WechatIMG525.png</span><span class="informations">2558×1094 571 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jiacai2050** [comment]
<p>It seems that <code>/repos/{owner}/{repo}</code> endpoint have <code>subscribers_count</code> field, but not in <code>/user/repos</code> endpoint.</p>
